### PR TITLE
Set transactions

### PR DESCRIPTION
**Pull request by emrojo**

### CODE REVIEW

**Comment by emrojo** [comment]
retest this pleas

**Comment by emrojo** [comment]
retest this please

File: app/resources/api/v1/set_transaction_resource.rb [hunk 1]
```diff
@@ -16,6 +16,21 @@ def meta(options)
         }
       end
 
+      before_create do
```
Comment (NikolausC) [inline app/resources/api/v1/set_transaction_resource.rb]: Please could this be commented to explain why it's necessary?

I think it's for whether a set is being created/already exists

Comment (emrojo) [reply app/resources/api/v1/set_transaction_resource.rb]: It will be the user to set as owner for the set if a new set is created in this transaction

**Comment by emrojo** [comment]
retest this please

**Comment by emrojo** [comment]
retest this please

**Pull request merged**